docs/.vuepress/config.js

```diff
@@ -144,12 +141,12 @@ module.exports = {
     }
   },
   plugins: [
-    // [
-    //   "@vuepress/google-analytics",
-    //   {
-    //     ga: "UA-51029217-12"
-    //   }
-    // ],
+    [
+      "@vuepress/google-analytics",
+      {
+        ga: "UA-51029217-12"
```

Comment:
is this optional, or is there a way to deny this?

Reply:
No, it is not. Do we want to make it optional?

Reply:
I would prefer it, but what do others think?